openmetadata-ui/src/main/resources/ui/.eslintrc.yaml

```diff
@@ -167,10 +167,14 @@ rules:
   '@typescript-eslint/explicit-function-return-type':
     - off
 
+  # disable the rule for all files
+  '@typescript-eslint/explicit-module-boundary-types':
+    - off
+
```

Comment:
Why we are disabling this rule?

Reply:
It will print tones of errors. We will slowly enable them if needed